lms/djangoapps/teams/static/teams/js/collections/base.js

```diff
@@ -0,0 +1,23 @@
+;(function (define) {
+    'use strict';
+    define(['common/js/components/collections/paging_collection'],
+        function(PagingCollection) {
+            var BaseCollection = PagingCollection.extend({
+                initialize: function(options) {
+                    PagingCollection.prototype.initialize.call(this);
+
+                    this.course_id = options.course_id;
+                    this.perPage = options.per_page;
+
+                    this.teamEvents = options.teamEvents;
+                    this.teamEvents.bind('teams:update', this.onUpdate, this);
+                    this.isStale = false;
+                },
+
+                onUpdate: function(event) {
+                    this.isStale = true;
+                }
```

Comment:
I was hoping that a helper method to do this would also be in this base class:

if (this.collection.isStale) {
+                    this.collection.fetch()
+                        .done(function() {
+                            self.collection.isStale = false;
+                            self.render();
+                        });
+                }

If you don't think it makes sense, leave it as-is.